modules/cnf-about-nic-support-irq-affinity-setting.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/cnf-low-latency-tuning.adoc
+
+:_content-type: CONCEPT
+[id="about_nic_support_irq_affinity_setting_{context}"]
+= About support of IRQ affinity setting
+
+Some IRQ controllers lack support for IRQ affinity setting and will always expose all online CPUs as the IRQ mask. These IRQ controllers effectively run on CPU 0.
+
+The following list by no means exhaustive are examples of drivers and hardware that Red Hat are aware lack support for IRQ re-balancing:
+
+* Some RAID controller drivers, such as `megaraid_sas`
+* Many non-volatile memory express (NVMe) drivers
+* Some LAN on motherboard (LOM) network controllers
+
+[NOTE]
+====
+The reason they do not support IRQ affinity setting, is probably associated with factors such as the type of processor, the IRQ controller, or the circuitry connections in the motherboard.
+====
```

Comment:
This is better now, but maybe "the driver uses managed_irqs" should be also mentioned here as possible cause. This is why RAID and NVMe drivers don't support IRQ re-balancing, and the this note is partially refering to them.